.pre-commit-config.yaml

```diff
@@ -47,10 +47,10 @@ repos:
     hooks:
       - id: yamllint
         args: ['--config-file', '.yamllint.yaml']
-#  - repo: https://github.com/mgedmin/check-manifest
-#    rev: "0.48"
-#    hooks:
-#      - id: check-manifest
+  - repo: https://github.com/mgedmin/check-manifest
+    rev: "0.48"
+    hooks:
+      - id: check-manifest
```

Comment:
https://github.com/mgedmin/check-manifest/blob/master/CHANGES.rst#048-2022-03-13

Git under 2.11 is not supported by revision 0.48 of `check-manifest`. Thus, this pre-commit hook systematically fails on doris (git 1.8.3), which is where I do most of my xscen development. Which means it affects at least 25% of the devs! ;)

Reply:
Right! I had forgotten about that! Have you asked Mourad if we could update git on our internal infra?

Reply:
Je viens de lui écrire.

Reply:
Maybe not, after all... See Pascal's message on Slack.